ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/OAuthRedirectExample/OAuthRedirectExample.pro

```diff
@@ -0,0 +1,45 @@
+TEMPLATE = app
+
+QT += core gui opengl xml network positioning sensors
+QT += qml quick
+CONFIG += c++11
+
+ARCGIS_RUNTIME_VERSION = 100.1
+include($$PWD/arcgisruntime.pri)
+
+HEADERS += OAuthRedirectExample.h \
+    OAuthRedirectHandler.h \
+    MyApplication.h
+
+SOURCES += main.cpp \
+    OAuthRedirectExample.cpp \
+    OAuthRedirectHandler.cpp \
+    MyApplication.cpp
+
+RESOURCES += OAuthRedirectExample.qrc
+
+win32 {
+    include (Win/Win.pri)
+}
+
+macx {
+    include (Mac/Mac.pri)
+    QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.11
+}
+
+ios {
+    include (iOS/iOS.pri)
+}
+
+android {
+    include (Android/Android.pri)
+}
+
+DEFINES +="CLIENT_ID=\"enter client id\""
+DEFINES +="URL_SCHEME=\"exampleapp\""
+
+win32|unix:!macx {
+    message("Using QtSingleApplication")
+    QT_SINGLEAPP_DIR = "$$PWD/../../../../../../../qt-solutions/qtsingleapplication"
```

Comment:
Might be good to add a comment that this needs to get updated to wherever they have cloned this repo